.circleci/config.yml

```diff
@@ -223,6 +266,14 @@ jobs:
       - image: circleci/python:3.6
         environment:
           TOXENV: py36-trinity-integration
+  py36-trinity-lightchain_integration:
+    <<: *common
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.6
+        environment:
+          TOXENV: py36-trinity-lightchain_integration
+          GETH_VERSION: v1.8.1
```

Comment:
Why such an old version?

Reply:
I think it's the latest supported by py-geth: https://github.com/ethereum/py-geth#installing-specific-versions-of-geth